utils/common/src/main/kotlin/FileMatcher.kt

```diff
@@ -39,13 +40,42 @@ class FileMatcher(
 ) {
     constructor(vararg patterns: String, ignoreCase: Boolean = false) : this(patterns.asList(), ignoreCase)
 
-    private val matcher = AntPathMatcher().apply {
-        setCaseSensitive(!ignoreCase)
+    companion object {
+        private val caseInsensitiveMatcher = AntPathMatcher().apply { setCaseSensitive(false) }
+        private val caseSensitiveMatcher = AntPathMatcher().apply { setCaseSensitive(true) }
```

Comment:
The above vars should be all caps, shouldn't they?

Reply:
Yes, but I have changed them to function references now, I think in this case I prefer lowercase.